packages/client/src/comp/states/Kansas.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import Select from 'muicss/lib/react/select'
+import { BaseInput } from '../util/Input'
+
+import { KansasInfo } from '../../common'
+import { SignatureBase, StatelessInfo, NoSignature } from './Base'
+import { Upload } from '../util/Upload'
+import { useControlRef } from '../util/ControlRef'
+import { TogglableDropdown } from '../util/TogglableDropdown'
+
+const options = {
+  driversLicenseNumber: 'Driver\'s License Number',
+  copyOfPhotoId: 'Copy of Photo ID',
+} as const
+
+export const Kansas = () => {
+  const idNumberRef = useControlRef<Select>()
+  const [idPhoto, setIdPhoto] = React.useState<string>()
+  const enrichValues = (baseInfo: StatelessInfo): NoSignature<KansasInfo> | null => {
+    const idNumber = idNumberRef.value() || undefined
+    const idType = idNumberRef.value() != null ? 'Number' : 'Image'
```

Comment:
If the user fills out both of these, `idType` logic will break.  We have to move the state in `packages/client/src/comp/util/TogglableDropdown.tsx` to a provider.